projects/solar/src/mppt.c

```diff
@@ -0,0 +1,72 @@
+#include "gpio.h"
+#include "mux.h"
+#include "spv1020_mppt.h"
+
+#define MUX_ENABLE \
+  { .port = GPIO_PORT_B, .pin = 1 }  // need to check hardward for correct pin
+#define MUX_OUTPUT \
+  { .port = GPIO_PORT_B, .pin = 1 }  // need to check hardward for correct pin
```

Comment:
Nit: maybe just have a comment saying "this pin is unused" or something like that